lib/assessment.rb

```diff
@@ -5,7 +5,10 @@ class Assessment
   # enumerable supplied.  Assume each entry implements a length
   # method that returns a Fixnum.
   def longest(enum)
-    # Your code goes here
+
+    ar = ['one','two','three','four','five']
+    ar.max_by(&:length)
+
```

Comment:
Nice! The only problem is that you were supposed to do it to `enum` instead of your own array.